pkg/monitoring/metrics/forklift-controller/migration_metrics.go

```diff
@@ -0,0 +1,110 @@
+package forklift_controller
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	api "github.com/konveyor/forklift-controller/pkg/apis/forklift/v1beta1"
+	"github.com/prometheus/client_golang/prometheus"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// Calculate Migrations metrics every 10 seconds
+func RecordMigrationMetrics(c client.Client) {
```

Comment:
what's the real benefit of migration that we need both migrations an plans? as plans provide more data (e.g type, destination) and both migrations and plans provide the status and provider?

Reply:
yes, i'm having the same doubts, I've comment on the metrics PR, but I think we can drop it.